src/modules/bonus/enrollment/BonusOnboarding_BuyTickets.tsx

```diff
@@ -4,15 +4,17 @@ import {EnrollmentOnboardingScreenProps} from '../../enrollment-onboarding/navig
 import {OnboardingScreenComponent} from '@atb/modules/onboarding';
 import {ThemedBonusTransaction} from '@atb/theme/ThemedAssets';
 import {ArrowRight} from '@atb/assets/svg/mono-icons/navigation';
-import {useEnrollmentOnboarding} from '@atb/modules/enrollment-onboarding';
+import {useNavigateToNextEnrollmentOnboardingScreen} from '../../enrollment-onboarding/use-navigate-to-next-onboarding-screen';
```

Comment:
```suggestion
import {useNavigateToNextEnrollmentOnboardingScreen} from '@atb/modules/enrollment-onboarding'
```

Should update to cleaner import